contents/tutorials/new-user-experiments.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Running Experiments on new users 
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-10
+featuredImage: ../images/tutorials/banners/new-user-experiments.png
+topics: ['experiments', 'feature flags']
+---
+
+- **Level:** Medium 🦔🦔
+- **Estimated reading time:** 10 minutes ☕️☕️
+
+Optimizing the initial experience of new users is critical for turning them into existing users. Products have a limited amount of time and attention from new users before they leave and churn. 
+
+One way PostHog helps create better new user experiences is through Experiments. Experiments help teams test changes and their effect on key metrics to create better products. 
+
+Experiments start with a proposed change and hypothesized impact. They then guide you through setting up the user groups and feature flags for testing. Finally, they provide analysis of results. 
+
+We’ll use all this to help us create a better new user experience.
+
+## Our experiment and what we need
+
+For our experiment, we want to see if a change to our onboarding process gets more new users to complete the first key event in our product, uploading a file. In our example, we have design and copy changes we’d to try to encourage new users to upload a file.
```

Comment:
I think we should give a bit more context about "the product" here to make it clear we're using a Dropbox-like service as an example.

Reply:
Got it